### ISSUE TITLE
Limit passes to certain weeks

### ISSUE THREAD

**Issue by nluedtke** [nluedtke/LineIt#11]

**Comment by nluedtke** [comment]
The idea of "passes" needs to be explained a bit more. Do you want the option to just recalculate stats based off only certain weeks? Or is there something more here?

**Comment by tnees** [comment]
Yeah, I want the algorithm to be able to isolate a stretch of time. So, we can keep using the entire season for the league in general, to make sure we're establishing good sample sizes for other teams' relative strengths, but be able to isolate certain periods of time.

So, for example, if I wanted to see how good the Cowboys were before Tony Romo and Dez Bryant got hurt, I'd want the algorithm to be able to run the entire league but ignore all but the specified games for the specified team (so eliminate all Cowboys games from week X-Y from the rating).

**Comment by nluedtke** [comment]
Alright, this is easy, you want to be able to ignore certain weeks for individual teams, but all other games for the league are still used in calculations.

**Comment by tnees** [comment]
Correct.
